projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader-config.types.ts

```diff
@@ -1,7 +1,7 @@
 import { InjectionToken } from '@angular/core';
 
 export type NgxSkeletonLoaderConfigTheme = {
-  // It enforces a combination of `fromRoot` styles with component `styles` attribute
+  /** It enforces a combination of `fromRoot` styles with component `styles` attribute */
```

Comment:
Unnecessary code changes. Could you revert that back, please?

Reply:
Changed the comment style, so you'll get a `tooltip` in your IDE.

<img width="695" alt="image" src="https://github.com/user-attachments/assets/34dc3ae0-4131-4c2c-abc0-b3f7e6f83dbc" />

Lemme know if you still want to have the comment style reverted

Reply:
Nah, let's keep it. Good catch, mate!